openedx/core/djangoapps/credentials/utils.py

```diff
@@ -18,7 +18,7 @@ def get_credentials_records_url(program_uuid=None):
     if base_url is None:
         return None
     if program_uuid:
-        return base_url + 'programs/{}/'.format(program_uuid)
+        return base_url + 'programs/{}/'.format(program_uuid, 'x')  # converts to hex
```

Comment:
You also can do "program_uuid.hex" I believe. Which might obviate the need for a comment.